teaser/data/output/modelicatemplate/AixLib/AixLib_Multizone

```diff
@@ -0,0 +1,377 @@
+    redeclare model AHUMod = AixLib.Airflow.AirHandlingUnit.AHU)
+% endif
+    "Multizone"
+    annotation (Placement(transformation(extent={{32,-8},{52,12}})));
+
+  AixLib.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(
+    calTSky=AixLib.BoundaryConditions.Types.SkyTemperatureCalculation.HorizontalRadiation,
+    computeWetBulbTemperature=false,
+    filNam=Modelica.Utilities.Files.loadResource(
+      "${weather}"))
+    "Weather data reader"
+    annotation (Placement(transformation(extent={{-82,30},{-62,50}})));
+
+  Modelica.Blocks.Sources.CombiTimeTable tableInternalGains(
+    tableOnFile=true,
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    tableName="Internals",
+    fileName=Modelica.Utilities.Files.loadResource(
+        "modelica://${mod_prj}/${bldg.name}/${bldg.file_internal_gains}"),
+    columns=2:${(3*get_n(bldg.thermal_zones))+1})
+    "Profiles for internal gains"
+    annotation (Placement(transformation(extent={{72,-42},{56,-26}})));
+
+  Modelica.Blocks.Sources.CombiTimeTable tableAHU(
+    tableOnFile=true,
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    tableName="AHU",
+    columns=2:5,
+    fileName=Modelica.Utilities.Files.loadResource(
+        "modelica://${mod_prj}/${bldg.name}/${bldg.file_ahu}"))
+    "Boundary conditions for air handling unit"
+    annotation (Placement(transformation(extent={{-64,-6},{-48,10}})));
+
+  Modelica.Blocks.Sources.CombiTimeTable tableTSet(
+    tableOnFile=true,
+    tableName="Tset",
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic,
+    fileName=Modelica.Utilities.Files.loadResource(
+        "modelica://${mod_prj}/${bldg.name}/${bldg.file_set_t}"),
+    columns=2:${get_n(bldg.thermal_zones)+1})
+    "Set points for heater"
+    annotation (Placement(transformation(extent={{72,-66},{56,-50}})));
+
+  Modelica.Blocks.Sources.Constant const[${get_n(bldg.thermal_zones)}](each k=0)
+    "Set point for cooler"
+    annotation (Placement(transformation(extent={{72,-90},{56,-74}})));
+
+equation
+  connect(weaDat.weaBus, multizone.weaBus) annotation (Line(
+      points={{-62,40},{-32,40},{-32,6},{34,6}},
+      color={255,204,51},
+      thickness=0.5));
+
+  connect(tableInternalGains.y, multizone.intGains)
+    annotation (Line(points={{55.2,-34},{48,-34},{48,-9}}, color={0,0,127}));
+
+  connect(tableAHU.y, multizone.AHU)
+    annotation (Line(points={{-47.2,2},{14,2},{33,2}}, color={0,0,127}));
+
+  connect(tableTSet.y, multizone.TSetHeat) annotation (Line(points={{55.2,-58},
+          {36.8,-58},{36.8,-9}}, color={0,0,127}));
+
+  connect(const.y, multizone.TSetCool) annotation (Line(points={{55.2,-82},{
+          34.6,-82},{34.6,-9}}, color={0,0,127}));
+
+  annotation (experiment(
+      StopTime=${modelica_info.runtime_simulation},
+      Interval=${modelica_info.interval_output},
+      __Dymola_Algorithm="${modelica_info.current_solver}"),
+      __Dymola_experimentSetupOutput(events=${get_true_false(modelica_info
```

Comment:
This is not the flag wo events. Add a seccond Flag to ModelicaInfo class or use this one right